packages/jbrowse-cli/src/commands/set-default-session.ts

```diff
@@ -0,0 +1,210 @@
+import { flags } from '@oclif/command'
+import { promises as fsPromises } from 'fs'
+import * as path from 'path'
+import JBrowseCommand from '../base'
+
+interface DefaultSession {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  [key: string]: any
+}
+
+interface Track {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  [key: string]: any
+}
+
+interface Config {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  assemblies?: { name: string; sequence: { [key: string]: any } }[]
+  configuration?: {}
+  connections?: unknown[]
+  defaultSession?: DefaultSession
+  tracks: Track[]
+}
+
+export default class SetDefaultSession extends JBrowseCommand {
+  static description = 'Set a default session with views and tracks'
+
+  static examples = []
+
+  static args = [
+    {
+      name: 'defaultSession',
+      required: false,
+      description: 'path to a default session setup',
+    },
+    {
+      name: 'location',
+      required: false,
+      description: 'path to JB2 installation',
+      default: '.',
+    },
+  ]
+
+  static flags = {
+    name: flags.string({
+      char: 'n',
+      description: 'Give a name for the default session',
+      default: 'New Default Session',
+    }),
+    view: flags.string({
+      char: 'v',
+      description:
+        'View type in config to be added as default session, i.e LinearGenomeView, CircularView, DotplotView.\nMust be provided if no default session file provided',
+    }),
+    viewid: flags.string({
```

Comment:
```suggestion
    viewId: flags.string({
```

viewId for consistent capitalization